modules/data-entry/src/main/frontend/src/questionnaireEditor/AnswerOptions.jsx

```diff
@@ -33,14 +37,37 @@ import { stringToHash } from "../escape.jsx";
 
 let AnswerOptions = (props) => {
   const { data, path, saveButtonRef, classes } = props;
-  let [ options, setOptions ] = useState(Object.values(data).filter(value => value['jcr:primaryType'] == 'lfs:AnswerOption').slice());
+  let [ options, setOptions ] = useState(Object.values(data).filter(value => value['jcr:primaryType'] == 'lfs:AnswerOption'
+                                                                               && !value.notApplicable
+                                                                               && !value.noneOfTheAbove).slice());
   let [ deletedOptions, setDeletedOptions ] = useState([]);
   let [ tempValue, setTempValue ] = useState(''); // Holds new, non-committed answer options
   let [ isDuplicate, setIsDuplicate ] = useState(false);
 
+  const notApplicable  = Object.values(data).find(option => option['jcr:primaryType'] == 'lfs:AnswerOption' && option.notApplicable);
+  const noneOfTheAbove = Object.values(data).find(option => option['jcr:primaryType'] == 'lfs:AnswerOption' && option.noneOfTheAbove);
+
+  let [ notApplicableOption, setNotApplicableOption ] = useState(notApplicable || {"value" : "None", "notApplicable" : false, "@path" : path + "/None"});
+  let [ noneOfTheAboveOption, setNoneOfTheAboveOption ] = useState(noneOfTheAbove || {"value": "None of the above", "noneOfTheAbove" : false,  "@path" : path + "/NoneOfTheAbove"});
+
+  const specialOptionsInfo = [
+    {
+      tooltip : "This option behaves as 'None' or 'N/A', and unselects/removes all other options upon selection.",
+      data : notApplicableOption,
+      setter : setNotApplicableOption,
+      label: "notApplicable"
+    },
+    {
+      tooltip : "This option behaves as 'None of the above', and unselects/removes all other options upon selection",
```

Comment:
This is not quite correct. `N/A` unselects ALL options, both predefined and free text, and disables the text input, but `None of the above` only unselects the predefined options, leaving free-text options selected, and keeps the text input enabled. I don't yet have a suggestion for improving the text, however.

But this makes me think of something else @marta- @k-pace:

If let's say `maxAnswers=3`, should "None of the above" count as one of those 3 options, leaving room for just 2 more custom ones? I would say no.

If `minAnswers=1`, should "None of the above" count as the minimum required answer? I'm not sure, I would say No, this is more like a marker that makes it explicit that none of the default options are good, but not an actual option by itself.

And another question: if `minAnswers=3`, should `N/A` make the requirement satisfied? I would say yes, the question doesn't apply, it allows the user to say it doesn't apply, so we can't force the user to say 3 times that it doesn't apply.

Reply:
The tooltip in the mockup posted on Jira is: "This option behaves as 'None of the above'. When selected, it removes all selections except those entered by the user in the input, if applicable". I would add an "existing" to clarify even further:
```suggestion
      tooltip : "This option behaves as 'None of the above'. When selected, it removes all existing selections except those entered by the user in the input, if applicable.",
```